src/function/uvc.c

```diff
@@ -138,6 +139,131 @@ struct {
 		.export = usbg_set_config_node_int,		        \
 	}
 
+int h2b(unsigned char c)
+{
+	/* check if char is a number */
+	if ((c >= '0') && (c <= '9'))
+		return c - '0';
+
+	/* check if char is upper case */
+	if ((c >= 'A') && (c <= 'Z'))
```

Comment:
Shouldn't be letters `c > 'F'` be illegal for a hex to bin conversion?

Reply:
Right. While implementing I just looked at the ASCII table. Good catch.

Reply:
I fixed that